aten/src/ATen/core/dispatch/Dispatcher.cpp

```diff
@@ -146,12 +146,12 @@ void Dispatcher::deregisterBackendFallbackKernel_(DispatchKey dispatchKey) {
 
 RegistrationHandleRAII Dispatcher::registerKernel(const OperatorHandle& op, DispatchKey dispatch_key, KernelFunction kernel) {
   // note: this doesn't need the mutex to protect the iterator because write operations on the list keep iterators intact.
-  return op.operatorIterator_->op.registerKernel(std::move(dispatch_key), std::move(kernel));
+  return op.operatorIterator_->op.registerKernel(dispatch_key, std::move(kernel));
```

Comment:
No longer need the move bc now implicitly converting to optional? or never needed the move?

Reply:
We never needed the move, dispatch keys are small